build/swivel/Interfaces.sol

```diff
@@ -9,7 +9,7 @@ import './Sig.sol';
 interface ISwivel {
   function initiate(Hash.Order[] calldata, uint256[] calldata, Sig.Components[] calldata) external returns (bool);
   function exit(Hash.Order[] calldata, uint256[] calldata, Sig.Components[] calldata) external returns (bool);
-  function cancel(Hash.Order calldata, Sig.Components calldata) external returns (bool);
+  function cancel(Hash.Order[] calldata) external returns (bool);
```

Comment:
the V3 branch did not have the update that let `cancel()` be called with an array of orders